src/sentry/migrations/0309_bit_int_for_org_and_project_id.py

```diff
@@ -0,0 +1,337 @@
+# Generated by Django 2.2.28 on 2022-07-25 20:49
+
+import django
+from django.db import migrations
+
+import sentry.db.models.fields.bounded
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+
+    # please don't preform database_operations as they already have been applied, they were added so the tests could run
```

Comment:
These were added so that these changes will propagate to self-hosted, ST, dev and ci environments